### PR TITLE
Combine Google Font challenges into one

### LINKED ISSUE #13471: [beta] Duplicated Google Font challenge
Challenge [use-a-google-font](http://beta.freecodecamp.com/en/challenges/applied-visual-design/use-a-google-font) has an issue.

Similarly to #13463, there are two versions of this challenge, one in [Basic CSS](http://beta.freecodecamp.com/en/challenges/basic-css/import-a-google-font) (the original one) and one in [Applied visual design](http://beta.freecodecamp.com/en/challenges/applied-visual-design/use-a-google-font) (this one was added with the new curriculum).

Actually both have a good description, I think the best thing would be to merge the two challenges to have only one on the subject, rather than just deleting one.

### PR DESCRIPTION
**Pull request by HKuz**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13471 

#### Description

The curriculum expansion added a challenge (in the Applied Visual Design section) about importing a Google Font. It covered the same concept as the existing "Use a Google Font" challenge in the Basic CSS section. This PR layers some additional information from the new challenge into the original Google Font challenge (and slightly tweaks the "How Fonts Degrade" challenge), and removes the new one.

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 1]
```diff
@@ -695,7 +695,12 @@
       "title": "Import a Google Font",
       "description": [
         "Now, let's import and apply a Google font (note that if Google is blocked in your country, you will need to skip this challenge).",
-        "First, you'll need to make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
+        "<a href='https://fonts.google.com/' target='_blank'>Google Fonts</a> is a great way to find free fonts to use in a project.",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: I recommend changing this line to:
>Google Fonts is a free library of web-safe fonts that you can use in your CSS by referencing the font's URL.

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 2]
```diff
@@ -695,7 +695,12 @@
...
+        "Once you have selected the preferred font, copy the link tag that is provided and insert it into the <code>head</code> of your page.",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: IMHO, this is misleading as this suggests they can choose any font for this challenge. I recommend removing this line.

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 3]
```diff
@@ -695,7 +695,12 @@
...
+        "Then, you can make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: I recommend changing this to:
>To import a Google Font, you can copy the font(s) URL from the Google Fonts library and then paste it in your HTML. For this challenge, we'll import the <code>Lobster</code> font. To do this, copy the following code snippet and paste it into the top of your code editor:
`<code>&#60;link href=\"https://fonts.googleapis.com/css?family=Lobster\" rel=\"stylesheet\" type=\"text/css\"&#62;</code>`

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 4]
```diff
@@ -695,7 +695,12 @@
...
+        "The font is available to include in your CSS by using the pattern:<br><code>font-family: FAMILY_NAME, GENERIC_NAME;</code>.",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: I recommend changing this to:
>Now you can use the `Lobster` font in your CSS by using `Lobster` as the FAMILY_NAME as in the following example:
`<br><code>font-family: FAMILY_NAME, GENERIC_NAME;</code>`

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 5]
```diff
@@ -695,7 +695,12 @@
...
+        "In this example, FAMILY_NAME would be <code>Lobster</code>. The GENERIC_NAME is optional, and is a fallback font in case the other specified font is not available. This is covered in the next challenge.",
+        "Family names are case-sensitive and needs to be wrapped in quotes if there is a space in the name. For example, you need quotes to use the <code>\"Open Sans\"</code> font, but not to use the <code>Lobster</code> font.",
         "Copy the following code snippet and paste it into the top of your code editor:",
         "<code>&#60;link href=\"https://fonts.googleapis.com/css?family=Lobster\" rel=\"stylesheet\" type=\"text/css\"&#62;</code>",
         "Remember, before you can apply styles, like a new font, to an element, you'll need to create a CSS rule.",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: Consider removing; seems unnecessary?

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 6]
```diff
@@ -695,7 +695,12 @@
...
+        "<a href='https://fonts.google.com/' target='_blank'>Google Fonts</a> is a great way to find free fonts to use in a project.",
+        "Once you have selected the preferred font, copy the link tag that is provided and insert it into the <code>head</code> of your page.",
+        "Then, you can make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
+        "The font is available to include in your CSS by using the pattern:<br><code>font-family: FAMILY_NAME, GENERIC_NAME;</code>.",
+        "In this example, FAMILY_NAME would be <code>Lobster</code>. The GENERIC_NAME is optional, and is a fallback font in case the other specified font is not available. This is covered in the next challenge.",
+        "Family names are case-sensitive and needs to be wrapped in quotes if there is a space in the name. For example, you need quotes to use the <code>\"Open Sans\"</code> font, but not to use the <code>Lobster</code> font.",
         "Copy the following code snippet and paste it into the top of your code editor:",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: Remove this line in favor of above.

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 7]
```diff
@@ -695,7 +695,12 @@
...
         "<code>&#60;link href=\"https://fonts.googleapis.com/css?family=Lobster\" rel=\"stylesheet\" type=\"text/css\"&#62;</code>",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: Remove this line in favor of above.

**Review by dhcodes — CHANGES_REQUESTED** [review]
@HKuz Thanks for doing this. Please see my comments; I think there are some ways the instructions could be shortened and improved. Let me know what you think.

**Comment by dhcodes** [comment]
@HKuz should've named your branch `oneGoogleFontToRuleThemAll` #missedopportunity 😆

**Comment by HKuz** [comment]
@dhcodes - I was *thisclose* to naming my branch that, then reconsidered because it seemed long. Haha - a missed opportunity indeed. 😞 

These comments are all good - I'm turning them now, and will update 👍

File: seed/challenges/01-responsive-web-design/basic-css.json [hunk 8]
```diff
@@ -695,11 +695,12 @@
       "title": "Import a Google Font",
       "description": [
         "Now, let's import and apply a Google font (note that if Google is blocked in your country, you will need to skip this challenge).",
-        "First, you'll need to make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
-        "Copy the following code snippet and paste it into the top of your code editor:",
+        "<a href='https://fonts.google.com/' target='_blank'>Google Fonts</a> is a free library of web-safe fonts that you can use in your CSS by referencing the font's URL.",
```
Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: @HKuz Great work! I made a mistake though. The fonts on Google Fonts aren't web-safe (my poor understanding of the terminology) so can you change "web-safe" to "web"

Comment (dhcodes) [inline seed/challenges/01-responsive-web-design/basic-css.json]: Otherwise it's good to go.

Comment (HKuz) [inline seed/challenges/01-responsive-web-design/basic-css.json]: Sounds good! Updating this now 👍

**Comment by dhcodes** [comment]
Tested and **LGTM**! Thanks for putting up with my suggestions :) 🌮 💎 :octocat:

**Review by dhcodes — APPROVED** [review]

**Comment by HKuz** [comment]
Thanks @dhcodes 🍻 🍀 :1st_place_medal: !!

**Comment by QuincyLarson** [comment]
@HKuz @dhcodes thanks for making this improvement. I agree - this is more streamline 😄

**Pull request merged**